content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.getCalendarType()'
+Description: 'Returns the calendar type of the Calendar instance'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.getCalendarType()` method of the `Calendar` class is used to return a String type of the calendar used.
+
+## Syntax
+
+```pseudo
+
+public String getCalendarType()
+
+```
+
+## Example
+
+The following example displays the type of the calendar.
+
+```java
+
+import java.util.*;
+
+public class MyClass {
+  public static void main(String[] args) {
+    //creating a Calendar object with specified date
```

Comment:
```suggestion
    // Creating a Calendar object with specified date
```

Reply:
Comments should begin with a capital and there should be a space before the first word